website/source/docs/providers/google/r/google_project.html.markdown

```diff
@@ -8,54 +8,69 @@ description: |-
+resource must have `roles/resourcemanager.projectCreator`. See the
+[Access Control for Organizations Using IAM](https://cloud.google.com/resource-manager/docs/access-control-org)
+doc for more information.
 
 ## Example Usage
 
 ```js
 resource "google_project" "my_project" {
-    id = "your-project-id"
-    policy_data = "${data.google_iam_policy.admin.policy_data}"
-}
-
-data "google_iam_policy" "admin" {
-  binding {
-    role = "roles/storage.objectViewer"
-    members = [
-      "user:evandbrown@gmail.com",
-    ]
-  }
+    project_id = "your-project-id"
+    org_id = "1234567"
+    services = ["compute_component", "storage-component-json.googleapis.com", "iam.googleapis.com"
 }
 ```
 
 ## Argument Reference
 
 The following arguments are supported:
 
-* `id` - (Required) The project ID.
-    Changing this forces a new project to be referenced.
+* `project_id` - (Optional) The project ID.
+    Changing this forces a new project to be created. If this attribute is not
+    set, `id` must be set. As `id` is deprecated, consider this attribute
+    required. If you are using `project_id` and creating a new project, the
+    `org_id` and `name` attributes are also required.
+
+* `id` - (Deprecated) The project ID.
+    This attribute was incorrectly exposed as configurable. If you used
+    this attribute, it would only work if you had also set the `project`
+    attribute of the Google `provider`, in which case that value would
+    override this one. This attribute will be removed in Terraform 0.9. You
+    should replace it with `project_id`.
 
-* `policy` - (Optional) The `google_iam_policy` data source that represents
-    the IAM policy that will be applied to the project. The policy will be
-    merged with any existing policy applied to the project.
+* `project_id` - (Required) The project ID.
+    Changing this forces a new project to be created.
 
-    Changing this updates the policy.
+* `org_id` - (Optional) The numeric ID of the organization this project belongs to.
+    This is required if you are creating a new project.
+    Changing this forces a new project to be created.
 
-    Deleting this removes the policy, but leaves the original project policy
-    intact. If there are overlapping `binding` entries between the original
-    project policy and the data source policy, they will be removed.
+* `name` - (Optional) The display name of the project.
+    This is required if you are creating a new project.
+
+* `services` - (Optional) The services/APIs that are enabled for this project.
+    For a list of available services, run `gcloud beta service-management list`
+
+* `skip_delete` - (Optional) If true, the Terraform resource can be deleted
+    without deleting the Project via the Google API.
+
+* `policy_data` - (Deprecated) This argument is no longer supported, and
+    an error will be thrown if it is present. It must be replaced with a 
+    `google_project_iam_policy` resource.
```

Comment:
I think we should probably soften the language around this, because it makes it sound like it no longer works even though it used to, which isn't (AFAIK) actually the case. It still works, but we suggest moving off it, and people will get a warning to move off it.